src/pages/api/pages/top-level.ts

```diff
@@ -0,0 +1,69 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { Portfolio } from "@prisma/client";
+
+import { SanitizedUser } from "../../../helpers/userHelper";
+
+import { constructHandler, InternalResponse } from "../../../helpers/errors";
+
+import type { UserSession } from "../../../helpers/userHelper";
+import { getPortfoliosByUser } from "../../../helpers/portfolioHelper";
+// import { createPortfolio } from "../../../helpers/portfolioHelper";
+
+import type { PortfolioWithPositions } from "../../../helpers/portfolioHelper";
+
+// TODO update
```

Comment:
add to todo here maybe? not required

Reply:
ah i mean when those two features are added, add it to the endpoint